src/commands/projects/remove.js

```diff
@@ -0,0 +1,66 @@
+const { BaseCommand } = require('@twilio/cli-core').baseCommands;
+
+class ProjectsRemove extends BaseCommand {
+  async run() {
+    await super.run();
+    this.projectId = this.args.project;
+    this.removeProjectStatus(this.projectId);
+    const verdict = await this.confirmRemoveProject();
+    this.callRemoveProject(this.projectId, verdict);
+    const configSavedMessage = await this.configFile.save(this.userConfig);
+    this.logger.info(configSavedMessage);
+  }
+
+  removeProjectStatus(projectId) {
+    const deleteProject = this.userConfig.getProjectById(projectId);
+    const activated = this.userConfig.getActiveProject();
+    if (!deleteProject) {
+      this.logger.error('The project "' + this.args.project + '" does not exist. Run "twilio projects:list" to see the list of configured projects.');
+      this.exit(1);
+    }
+    if (activated.id === projectId) {
+      this.logger.warn('Removing the active project. Run "twilio projects:use" to set another project as active.');
+    }
+    if (this.userConfig.projects.length === 1) {
+      this.logger.warn('Removing last project. Run "twilio projects:add" to add another project.');
+    }
+  }
+
+  callRemoveProject(projectId, verdict) {
+    const deleteProject = this.userConfig.getProjectById(projectId);
+
+    if (verdict === true) {
+      this.userConfig.removeProject(deleteProject);
+      this.logger.info('Deleted ' + projectId + '. The key still exists in the Twilio Platform.');
```

Comment:
Couldn't we just delete it for them?

Reply:
I think we should _attempt_ to delete the key (after asking the user) and also remove the key from the local key store.